content/en/tracing/enable/_index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Enable trace collection in Datadog
```

Comment:
the title is not entirely true since we don't enable trace collection in Datadog, we enable trace collection within the Datadog Agent, or through an other way.